examples/cluster-dns/images/frontend/client.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
From https://github.com/GoogleCloudPlatform/kubernetes/pull/7672/files#r30158601: "We regex all 2015->2014 before we compare with the boilerplate"
@eparis